content/posts/scientific-python/translations/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Setting up and managing translations for Scientific Python projects"
+date: 2024-08-08
+draft: false
+tags:
+  [
+    "Scientific-Python",
+    "translations",
+    "inclusion",
+    "CZI",
+  ]
+description: "Setting up and managing translations for Scientific Python projects."
+displayInList: true
+author: ["Melissa Mendonça", "Albert Steppi", "Tania Allard", "Ralf Gommers", "Stéfan van der Walt", "Jarrod Millman"]
```

Comment:
You don't need to include me as an author; you did all the work!

Reply:
+1 same here. I think the author of this blog post is either just you Melissa, or you and Albert if it was a collaboration on the content.